### PR TITLE
Third task: should be able to click on a country and get a list of country's leagues

### PR DESCRIPTION
**Pull request by otherdcn**
The two commits to consider:

1. First commit implements the single responsibility principle, as discussed.
2. Second commit accomplishes the third task.

I simply followed same structure, as the CountriesHelper, of creating the "leagues" API endpoint, via the LeaguesHelper object, by inheriting from the ApplicationHelper::APIFootball.

What do you think?

### CODE REVIEW

File: app/controllers/countries_controller.rb [hunk 1]
```diff
@@ -7,13 +7,18 @@ def index
     @countries
   end
 
+  def show
```
Comment (javiros) [inline app/controllers/countries_controller.rb:10]: we would probably need to start adding ~controller~ requests specs

Comment (javiros) [reply app/controllers/countries_controller.rb:10]: https://rspec.info/features/6-0/rspec-rails/request-specs/request-spec/#:~:text=Request%20specs%20provide%20a%20thin,if%20you%20have%20set%20config.

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,2 +1,8 @@
 module ApplicationHelper
+  class APIFootball
```
Comment (javiros) [inline app/helpers/application_helper.rb:2]: same here, specs needed

**Review by javiros — APPROVED** [review]
A few comments around missing specs, I thnk we should stop developing new features and start adding tests

**Comment by otherdcn** [comment]
Well received. Will look into them it and get them done.

**Pull request merged**